src/components/Button.js

```diff
@@ -21,11 +25,41 @@ const Button = React.createClass({
 
   getDefaultProps () {
     return {
+      isActive: false,
       primaryColor: StyleConstants.Colors.PRIMARY,
       type: 'primary'
     };
   },
 
+  _renderButtonContent () {
+    const spinnerStyles = {
+      verticalAlign: 'initial',
+      marginTop: -6,
+      marginBottom: -5,
+      marginLeft: -5,
+      marginRight: -5,
+      padding: !this.props.actionText && this.props.children ? 3 : 0
+    };
+
+    const actionTextStyles = {
+      display: 'inline-block',
+      paddingLeft: 10
+    };
```

Comment:
Is declaring the styles variable on lines 35 & 52 okay or is there a better place to declare this once?